rust/osm2lanes/src/road/lane.rs

```diff
@@ -33,8 +35,14 @@ impl Lane {
     pub fn width(&self, locale: &Locale) -> Metre {
         match self {
             Lane::Separator { markings } => markings.width(locale),
-            Lane::Travel { designated, .. } => locale.default_width(designated),
-            _ => Lane::DEFAULT_WIDTH,
+            Lane::Travel {
+                width, designated, ..
+            } => width.unwrap_or_else(|| locale.travel_width(designated)),
+            // TODO: parking different from travel?
```

Comment:
100% yes, parking lanes are way more narrow than travel lanes in the UK a tleast